src/test/ui/issues/issue-66065.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
Can you give the test a more informative file name + add a comment describing what was wrong?

Reply:
Moved it with all the other formatting tests.